lib/qa_utils.cc

```diff
@@ -141,7 +141,7 @@ volk_type_t volk_type_from_string(std::string name) {
             type.is_signed = false;
             break;
         default:
-            throw;
+            throw "no such type!";
```

Comment:
I like this: let's feed back -something- useful! I'd like to augment this with the value of `name[i]` somehow: make it clear what the bad parsing type is; could be useful as feedback to the user as to what to look for to fix.

Reply:
this still

Reply:
This one.

Reply:
https://stackoverflow.com/questions/12261915/how-to-throw-stdexceptions-with-variable-messages How do you throw a variable name[i]?

Reply:
gotcha. let me figure that one out & get back shortly ...

Reply:
```
throw std::runtime_error (std::string ("Error: no such type: '") + name[i] + "'");
```

Reply:
> throw std::runtime_error("check(): Hidden node count of " + std::to_string(N_size_file) + " exceeds maximum allowed value of " + std::to_string(_N_MAX));

So, something like that. Found here: http://www.cplusplus.com/forum/beginner/57892/#msg311957
Is it safe? Or not? And can you do std::to_string in a loop??

Reply:
In our specific case here, `name[i]` is a `char`, and `std::string::operator+` covers the case of `std::string + char` ... so nothing special required in this specific case. If the argument were not a `char` then we'd have to do something else like `std::to_string`.

Reply:
Okay, thanks. Hope we will merge this in 30 minutes ;)